internal/distro/fedora32/distro.go

```diff
@@ -267,6 +267,11 @@ func sources(packages []rpmmd.PackageSpec) *osbuild.Sources {
 		fileSource := osbuild.FileSource{
 			URL: pkg.RemoteLocation,
 		}
+		if pkg.Secrets == "org.osbuild.rhsm" {
```

Comment:
Is there any usecase for Fedora and rhsm?

Reply:
Very unlikely, but it should be possible to setup private mirror (e.g. using Satellite or Pulp) with Fedora content and require subscription. Anyway it is useful for the test case I introduced.